Patches.py

```diff
@@ -1622,8 +1726,9 @@ def set_entrance_updates(entrances):
     shuffle_messages.shop_item_messages = []
 
     # kokiri shop
+    shop_locations = world.get_region('KF Kokiri Shop').locations
     shop_objs = place_shop_items(rom, world, shop_items, messages,
-        world.get_region('KF Kokiri Shop').locations, True)
+        shop_locations[1:], True)
```

Comment:
Doesn't this mean you're skipping the first location in the list? Why are you doing that?

Reply:
The PR adds a freestanding item to the Kokiri Shop, replacing the invisible blue rupee behind the corner. It would probably be better to filter by locations of type `Shop` here.

Reply:
Ah right. Yeah filtering by shop would be cleaner.

Reply:
Think I did this right.